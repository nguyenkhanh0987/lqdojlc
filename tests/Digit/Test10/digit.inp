100
374062661 387555406
891676055 899856635
741285286 852753335
230255424 528478638
795633849 872184985
775661766 949165739
27783516 462763764
31858989 716295770
113486883 745004823
734061941 804611387
966001489 985184497
541205563 822200582
934736204 974830576
555858969 578258350
995181962 996788898
66692774 454526237
806300344 954764585
844694141 977480464
167817892 945946584
75852369 975122410
937078489 939945568
182439170 394776640
251633608 709215191
686105636 956406954
381054481 732914732
214461465 221229705
114251694 288494906
264205664 996025031
667621524 965698477
675918705 759479325
28924860 845484516
905785511 908226116
2634360 865900206
594104265 615105781
649291260 670250464
561848534 769325894
599857101 714119414
139394058 478940330
566175850 571965466
749886200 971959595
249213007 294894149
619563748 949957320
826099722 939747233
481497125 768944182
938722485 980505440
592766271 676171377
246345354 585609851
113860733 121138017
721399728 903656508
528337241 665564816
495841325 586612215
609809561 793350418
279062423 640626400
264879905 955380968
792591750 857526990
98991821 541721475
394180890 482798014
40597511 691156573
291544594 635657816
884750801 972018224
775901603 784594713
473154368 580901832
168523982 267721867
97895848 544811119
197500902 652831496
124276048 297195730
68511132 107246729
25707868 975053162
211116658 269718172
532934684 963985001
562553815 948005396
239602082 956189157
867431004 960750529
786772575 843727521
996944653 998623270
574884235 776665082
759836130 847615168
387638561 701057652
819238087 891940857
265792729 717757779
489008031 731372930
258596264 433053089
96432133 815340022
496089116 800167053
676774408 768831509
582741236 933714181
668163954 831845762
114004215 414438384
197151460 359892312
946452513 995754469
148479800 442824515
761117625 832220267
226817369 586841387
198324290 556115037
440584493 570051804
679513821 771026205
540840246 550377022
713988283 787877204
494661529 771666906
367738521 560407732
