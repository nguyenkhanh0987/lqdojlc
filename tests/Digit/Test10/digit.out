2807851
1046951
9057471
44937846
4407590
12128342
75497229
104557871
88252990
7300731
1761220
28513746
2506286
3184218
155580
66194433
9002775
9102548
93927515
113594771
27987
36322513
62110984
20157999
41463696
622382
31858126
80983102
23620418
7110376
114996763
15169
120257757
1699994
2346958
21056347
10783542
55960582
805997
16618749
10064289
26799491
7461525
30271644
2758311
7712712
49816842
788641
15252168
15106779
10112707
17633443
48327491
77068170
3627441
68778311
11312814
100927551
45051592
5000429
1165280
13544290
17777576
69467302
62332843
32840381
9446018
124300712
8932708
38336433
33986718
81845669
5999426
3597022
100311
20034186
7229218
38547352
5771494
59640110
25687574
27609005
97664134
31370341
8048532
30197469
14689182
51268094
24864252
4239557
48769110
6202392
52901632
51447265
16660337
8102083
955541
6339594
27737859
25580598
